### PR TITLE
WMCO 3.1.0 RNs

### PR DESCRIPTION
**Pull request by codyhoag**
In preparation for the WMCO release on 2021/09/21.

Preview: https://deploy-preview-35723--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-3-x#wmco-3-1-0

### CODE REVIEW

File: windows_containers/windows-containers-release-notes-3-x.adoc [hunk 1]
```diff
@@ -17,6 +17,39 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-3-1-0"]
+== Release notes for Red Hat Windows Machine Config Operator 3.1.0
+
+Issued: 2021-09-21
+
+The WMCO 3.1.0 is now available with bug fixes and a new feature. The components of the WMCO were released in link:https://access.redhat.com/errata/RHBA-2021:3215[RHBA-2021:3215].
```
Comment (bmcelvee) [inline windows_containers/windows-containers-release-notes-3-x.adoc:25]: Would it make sense to spell out WMCO on first reference here?

File: windows_containers/windows-containers-release-notes-3-x.adoc [hunk 2]
```diff
@@ -17,6 +17,39 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
...
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* VMware vSphere
+
+BYOH Windows instances are supported for the following platforms with user-provisioned infrastructure when the `platform: none` field is set in the `install-config.yaml` file:
+
+* VMware vSphere
+* bare metal
+
+For more information on how to configure BYOH Windows instances, see xref:../windows_containers/creating_windows_machinesets/byoh-windows-instance.adoc#byoh-windows-instance[Configuring BYOH Windows instance].
```
Comment (bmcelvee) [inline windows_containers/windows-containers-release-notes-3-x.adoc:46]: This link is broken

**Comment by bmcelvee** [comment]
A few notes, otherwise LGTM!

File: windows_containers/windows-containers-release-notes-3-x.adoc (hunk 1 above)
Comment (codyhoag) [reply windows_containers/windows-containers-release-notes-3-x.adoc:25]: The intro of the release notes gives the full name, so I think the shorthand reference here should be safe.

File: windows_containers/windows-containers-release-notes-3-x.adoc (hunk 2 above)
Comment (codyhoag) [reply windows_containers/windows-containers-release-notes-3-x.adoc:46]: The link will work once the referenced content is merged :+1:

File: windows_containers/windows-containers-release-notes-3-x.adoc [hunk 3]
```diff
@@ -17,6 +17,44 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
...
+
+You can now add an existing Windows instance to an {product-title} cluster as a compute node. This requires creating a config map in the WMCO namespace.
+
+BYOH Windows instances are supported for the following platforms with installer-provisioned infrastructure:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* VMware vSphere
+
+BYOH Windows instances are supported for the following platforms with user-provisioned infrastructure when the `platform: none` field is set in the `install-config.yaml` file:
```
Comment (jrvaldes) [inline windows_containers/windows-containers-release-notes-3-x.adoc:41]:
```suggestion
BYOH Windows instances are supported in a User-Provisioned Infrastructure (UPI) with the `platform: none` field set in the `install-config.yaml` file, for the following platforms:
```

File: windows_containers/windows-containers-release-notes-3-x.adoc [hunk 4]
```diff
@@ -17,6 +17,44 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
...
+
+* VMware vSphere
+* bare metal
```
Comment (jrvaldes) [inline windows_containers/windows-containers-release-notes-3-x.adoc:44]: s/bare metal/BareMetal

**Review by jrvaldes — COMMENTED** [review]
@codyhoag  Thanks for working on this, added minor suggestions. PTAL

File: windows_containers/windows-containers-release-notes-3-x.adoc (hunk 4 above)
Comment (codyhoag) [reply windows_containers/windows-containers-release-notes-3-x.adoc:44]: This title casing is consistent with our other [bare metal](https://docs.openshift.com/container-platform/4.8/installing/installing_bare_metal/installing-bare-metal.html) docs for OCP.

File: windows_containers/windows-containers-release-notes-3-x.adoc (hunk 3 above)
Comment (codyhoag) [reply windows_containers/windows-containers-release-notes-3-x.adoc]: Thanks, @jrvaldes. I updated this wording a bit :+1:. In our docs, we avoid using the "UPI" acronym, as it's more of an internal thing that we don't frequently broadcast out.

**Pull request merged**